src/screens/SendTokens.js

```diff
@@ -289,14 +345,24 @@ class SendTokens extends React.Component {
   }
 
   /**
-   * Executed when user clicks to add another token to this transaction  
-   * Checks if still have a known token available that is not selected yet  
+   * Executed when user clicks to add another token to this transaction
+   * Checks if still have a known token available that is not selected yet
    * Create a new child reference with this new token
    */
   addAnotherToken = () => {
     if (hathorLib.wallet.isHardwareWallet()) {
-      $('#notSupported').modal('show');
-      return;
+      if (!version.isLedgerCustomTokenAllowed()) {
+        // Custom token not allowed for this Ledger version
+        $('#notSupported').modal('show');
```

Comment:
This modal will show the following message: `Unfortunately this feature is currently not supported when using a hardware wallet. [...]`

Shouldn't we show a different message in this case? Saying that the Ledger version is not supported, instead of saying that hardware wallet is not supported.